src/Relation/Binary/Consequences.agda

```diff
@@ -164,3 +164,9 @@ module _ {P : REL A B ℓ₁} {Q : REL B A ℓ₂} where
 
   flip-Connex : Connex P Q → Connex Q P
   flip-Connex f x y = Sum.swap (f y x)
+
+module _ {a b r} {A : Set a} {B : Set b} {R : REL A B r} where
+     dec⟶irrel′ : Decidable R → Irrelevant′ R
```

Comment:
You can drop `{a b} {A : Set a} {B : Set b}` thanks to [the variables](https://github.com/agda/agda-stdlib/blob/5741c80877e7d3ffcee9a842804ccb794719d3d3/src/Relation/Binary/Consequences.agda#L21).